src/main/webapp/app/course/manage/course-update.component.ts

```diff
@@ -252,6 +259,20 @@ export class CourseUpdateComponent implements OnInit {
         }
 
         const course = this.courseForm.getRawValue();
+        const communicationEnabled = !!this.courseForm.get('informationSharingSettings.communicationEnabled')?.value;
+        const messagingEnabled = !!this.courseForm.get('informationSharingSettings.messagingEnabled')?.value;
```

Comment:
Is it not possible to bind attributes directly to the form using a `[ngModel]` directive for those two values? That would reduce accidental breakage in case the form element changes in the HTML.

Reply:
Done in ab6242150d41f8dc69cdfca1e782799d72cd2e1a :)